src/server/controllers/Grbl/GrblController.js

```diff
@@ -1530,6 +1530,15 @@ class GrblController {
                 ];
                 this.command('gcode', commands);
             },
+            'laserpower:change': () => {
+                const [power = 0, maxS = 1000] = args;
+                const commands = [
+                    // https://github.com/gnea/grbl/wiki/Grbl-v1.1-Laser-Mode
+                    // The laser will only turn on when Grbl is in a G1, G2, or G3 motion mode.
+                    'G1F1 M3 S' + ensurePositiveNumber(maxS * (power / 100))
```

Comment:
Two other comments:

1) You only actually need to send the S command portion to update the power. 
2) I would add a trailing debounce to the slider handler so that it only is run when the user is finished sliding and not multiple times through it moving.  You can look at using the lodash debounce since it's already installed and we use it in a few other places.
![lots_of_calls](https://user-images.githubusercontent.com/22222507/179224116-f4ac2706-d399-4968-9ffc-2064b1150f4a.png)